tests/test_generators/input/rustgen_dataframe.yaml

```diff
@@ -0,0 +1,111 @@
+id: https://w3id.org/linkml/examples/dataframe_conversion
+name: dataframe_conversion
+prefixes:
+  ex: https://w3id.org/linkml/examples/dataframe_conversion/
+  linkml: https://w3id.org/linkml/
+imports:
+  - linkml:types
+default_prefix: ex
+default_range: string
+
+classes:
+  NestedRecord:
+    description: Nested object inlined in lists and maps
```

Comment:
I'd like to see descriptions on each of the slots, particularly the ones that correspond to the inlined forms.
It doesn't only have to do that, but especially with the names changing it feels a little easier to follow the slot to the corresponding logic.

At this point I am able to use a very similar model in my test.  I can approve this as-is, the comment are more to think on if we would be able to share this file after my next round of PRs for Panderagen.

Reply:
Yes, one of the PRs I put up soon will have my version of your schema as a yaml under the tests/test_generators/panderagen/inputs/ let's discuss if/how to further merge there.